### PR TITLE
fix(dependencies): replace react-native-app-intro with rn-app-intro

### PR DESCRIPTION
**Pull request by Ernoff**
Replace buggy package with newly bundled and published one compatible with React v.16.^

### CODE REVIEW

**Comment by lex111** [comment]
What are the differences from #399?

**Comment by Ernoff** [comment]
#399 had github repo in the package.json. So @andrewda asked me to publish it to npm and use the published package instead.

Can it be closed? I leave the answer to you guys

File: .all-contributorsrc [hunk 1]
```diff
@@ -371,6 +371,16 @@
         "doc",
         "tool"
       ]
+    },
+    {
+      "login": "Ernoff",
```
Comment (housseindjirdeh) [inline .all-contributorsrc]: :tada:

**Comment by Ernoff** [comment]
That is very fine with me!!

File: src/auth/screens/login.screen.js [hunk 2]
```diff
@@ -223,8 +223,8 @@ class Login extends Component {
     return (
       <View style={styles.browserLoader}>
         <Text style={styles.browserLoadingLabel}>
-          {this.state.loaderText}
-        </Text>
+          {' '}{this.state.loaderText}{' '}
+        </Text>{' '}
```
Comment (chinesedfan) [inline src/auth/screens/login.screen.js]: @Ernoff @housseindjirdeh Why are there so many {' '} additions in this file?

Comment (Ernoff) [reply src/auth/screens/login.screen.js]: Must be keyboard glitch. Apologies

Comment (Ernoff) [reply src/auth/screens/login.screen.js]: #410 

This cleans it up

**Pull request merged**